config-connector/solutions/iam/kpt/pubsub-topic/0-namespace.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
```

Comment:
There is no need to set the namespace if we don't create a new project or if we are not required to reference to a project. And in this solution, a project ID is not required. So we will not create a new namespace, but use the default namespace.